### PR TITLE
Added footer

### PR DESCRIPTION
**Pull request by owenkcraig**
Got that sweet footer in there!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,5 +13,8 @@ <h1>Check out this cool frog!</h1>
         <img src="https://external-preview.redd.it/aoOxi1firITL5t_svWtPjGSY0jgGgaLXibDcogLJX8Q.jpg?auto=webp&s=652e931cf9f51c2dcf16278de318d5fd2b877875" alt="frog with umbrella">
         <h2>I like this frog.</h2>
     </main>
+    <div>
```
Comment (kcheverie) [inline index.html]: Footer looks good, but let's make sure to use a semantic `<footer>` element.

**Review by kcheverie — CHANGES_REQUESTED** [review]
Just one small fix and then it's good to go.

File: index.html (hunk 1 above)
Comment (owenkcraig) [reply index.html]: Oh yeah! Great suggestion, you're so smart!

**Review by kcheverie — APPROVED** [review]
Looks great! thanks for the fast turnaround.

**Pull request merged**